app/models/katello/erratum.rb

```diff
@@ -150,21 +143,6 @@ def self.list_filenames_by_clauses(repo, clauses)
           where(statement).pluck(:filename)
     end
 
-    def module_streams
```

Comment:
Was this deletion intentional, or just some issue during resolving conflicts?

Reply:
Probably some strange conflict resolve. Not something I intended to do anyway :blush: